custom/enikshay/integrations/ninetyninedots/repeater_generators.py

```diff
@@ -132,6 +181,103 @@ def handle_failure(self, response, person_case, repeat_record):
             )
 
 
+@RegisterGenerator(NinetyNineDotsAdherenceRepeater, 'case_json', 'JSON', is_default=True)
+class AdherencePayloadGenerator(BasePayloadGenerator):
```

Comment:
not sure if/where these are documented? could be nice for these each to have a doc string indicating what purpose they are serving.